accounts/accounts_cli.go

```diff
@@ -0,0 +1,328 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/codegangsta/cli"
+	"github.com/howeyc/gopass"
+)
+
+// Represents the user running the audit on the accounts that they have access to via permissons
+type admin struct {
+	host   string
+	client *http.Client
+	User   struct {
+		Id    string `json:"userid"`
+		Name  string `json:"username"`
+		Token string `json:"-"`
+	} `json:"administrator"`
+	Accounts []*Account `json:"administeredAccounts"`
+}
+
+// Account details that we will audit
+type Account struct {
+	Id      string `json:"userid"`
+	Profile struct {
+		FullName string `json:"fullName"`
+		Patient  struct {
+			Bday string `json:"birthday"`
+			Dday string `json:"diagnosisDate"`
+		} `json:"patient"`
+	} `json:"patientProfile"`
+	Perms      interface{} `json:"permissons"`
+	LastUpload string      `json:"lastupload"`
+}
+
+var (
+	prodHost    = "https://api.tidepool.io"
+	stagingHost = "https://staging-api.tidepool.io"
+	develHost   = "https://devel-api.tidepool.io"
+	localHost   = "http://localhost:8009"
+)
+
+func main() {
+
+	app := cli.NewApp()
+
+	app.Name = "Accounts-Managment"
+	app.Usage = "Allows the bulk management of tidepool accounts"
+	app.Version = "0.0.1"
+	app.Author = "Jamie Bate"
+	app.Email = "jamie@tidepool.org"
+
+	app.Commands = []cli.Command{
+
+		//e.g. audit -u admin@place.org -e prod
+		//e.g. audit -u admin@place.org -e prod -r ./audit_admin@place.org_accounts_prod_2015-07-01T03:00:58Z.txt
+		{
+			Name:      "audit",
+			ShortName: "a",
+			Usage:     `audit all accounts that you have permisson to access`,
```

Comment:
We might want to standardize on double-quotes (rather than back-quotes) except for the (hopefully) rare cases where we need raw string literals.